modules/ocf_ceph/manifests/init.pp

```diff
@@ -0,0 +1,26 @@
+# Base class for ceph
+
+class ocf_ceph {
+  require ocf::packages::ceph
+
+  $ceph_fsid = lookup('ceph::fsid')
+
+  $ceph_mons_list = lookup('ceph::mons')
+  $ceph_mons = $ceph_mons_list.join(',')
+  $ceph_mon_ips = $ceph_mons_list.map
+    |$node| { ldap_attr($node, 'ipHostNumber') }.join(',')
+
+  $ip_addr = $::ipaddress
+
+  file { '/etc/ceph/ceph.conf':
+    content => template('ocf_ceph/ceph.conf.erb'),
+    require => Package['ceph'],
+  }
+
+  file { '/var/lib/ceph':
```

Comment:
this seems better suited for the `mon.pp` file since nothing here actually uses this.

Reply:
This is needed by osd.pp.